otter/assign/notebook_transformer.py

```diff
@@ -203,7 +203,7 @@ def _get_transformed_cells(self, cells):
                     solution_has_md_cells, prompt_insertion_index = False, None
 
                 elif block_type is BlockType.SOLUTION:
-                    if not has_prompt and solution_has_md_cells:
+                    if not has_prompt and solution_has_md_cells and question.manual:
```

Comment:
I ended up modifying this if statement, does this seem correct?

Reply:
LGTM